numpyro/distributions/patch.py

```diff
@@ -1,4 +1,5 @@
 import scipy
+import scipy.stats
```

Comment:
this import is used to avoid the error "AttributeError: module 'scipy' has no attribute 'stats'" which happens in case
```
import scipy
scipy.stats.uniform.pdf(0.5)  <-- error
```
In particular, we patch `scipy.stats._distn_infrastructure.rv_frozen.__call__`, so we have to import scipy.stats